resources/views/partials/searchform.blade.php

```diff
@@ -1,7 +1,7 @@
-<form role="search" method="get" class="search-form" action="{{ esc_url( home_url( '/' ) ) }}">
+<form role="search" method="get" class="search-form" action="{{ esc_url(home_url( '/') ) }}">
```

Comment:
👀

Reply:
See commit d93964b -- this was already done